ansible/roles/nginx/templates/nginx.conf.j2

```diff
@@ -83,6 +89,8 @@ http {
         proxy_ssl_verify off;
         proxy_ssl_session_reuse on;
 
+        proxy_set_header OW-TID $ow_tid;
```

Comment:
can you add a comment here that clarifies whether it's possible for a client to override this internally set header?

Reply:
Can we use a known header name: `X-Request-ID` instead of a custom header ?